asapdiscovery-data/asapdiscovery/data/util/utils.py

```diff
@@ -341,6 +341,126 @@ def cdd_to_schema(cdd_csv, out_json=None, out_csv=None):
     return compounds
 
 
+def cdd_to_schema_v2(
```

Comment:
not sure if you're already planning on updating this, but as it stands this will break my pipeline. I think I'd need:
* ability to pass a list of values to `target_prop`
* boolean flags for also getting CI values, in-range values, and uncertainty values (see keys I have in `reqd_cols` in the original version of this function)

the second bullet could be handled also as lists, so some keys in `target_prop` can require the extra information while some may not need it

Reply:
I think we can just let these live side by side for now, and then we can reconcile them in the future?